mon-pix/app/components/challenge-item-qrocm.js

```diff
@@ -10,6 +10,8 @@ import jsyaml from 'js-yaml';
 class ChallengeItemQrocm extends ChallengeItemGeneric {
   @service intl;
 
+  answerValues = [];
```

Comment:
Ce n'est pas un tableau mais un objet, on devrait le changer par 
```suggestion
  answerValues = {};
```
De plus, sans initialisation avec tous les champs, on ne peut détecter un remplissage partiel.
Il faudrait qu'à l'init, on est un objet de ce genre :
```
answerValues = { field1: '', field2: '', field3: '' };
```